RadarSDK/CLLocation+Radar.m

```diff
@@ -8,17 +8,23 @@
 @import Foundation;
 #import "CLLocation+Radar.h"
 
+const double DEGREE_EPSILON = 0.00000001;
```

Comment:
I believe obj C has this `DBL_EPSILON` global const that we can use, maybe we can consider its use here? I think it can be imported with `float.h`.

Reply:
`DBL_EPSILON` is approximately 2.2e-16 which is too precise I think, it would give similar results to just using the `!=` operator like we did before